subprojects/base-services/src/main/java/org/gradle/internal/service/scopes/Scopes.java

```diff
@@ -16,11 +16,55 @@
 
 package org.gradle.internal.service.scopes;
 
+/**
+ * @see ServiceScope
+ */
 public interface Scopes {
+    /**
+     * These services are reused across builds in the same process while the Gradle user home directory remains unchanged.
+     * The services are closed when the Gradle user home directory changes.
+     *
+     * <<p>{@link Scope.Global} and parent scope services are visible to {@link UserHome} scope services, but not vice versa.</p>
+     */
     interface UserHome extends Scope.Global {}
+
+    /**
+     * These services are reused across build invocations when in continuous mode.
+     * They are closed at the end of the build session.
+     *
+     * <p>{@link UserHome} and parent scope services are visible to {@link BuildSession} scope services, but not vice versa.</p>
+     */
     interface BuildSession extends UserHome {}
+
+    /**
+     * These services are recreated when in continuous mode and shared across all nested builds.
```

Comment:
:o: I'm not 100% sure about this one. It feels a bit confusing: those services are created _even without continuous mode_.

Reply:
If it's inaccurate, can you suggest a different phrasing?